torch/_inductor/utils.py

```diff
@@ -1509,3 +1515,142 @@ def is_gpu(device: str):
 def device_need_guard(device: str):
     assert isinstance(device, str)
     return is_gpu(device)
+
+
+@functools.lru_cache(None)
+def aoti_eager_cache_dir():
+    return Path.home() / ".cache" / "torch" / "aoti_eager"
+
+
+def load_aoti_eager_cache(
+    ns: str, op_func_name: str, op_overload_name: str, device_type: str
+):
+    device_kernel_cache = aoti_eager_cache_dir() / ns.lower() / device_type.lower()
+    op_overload_name = op_overload_name if op_overload_name else "default"
+    op_conf = device_kernel_cache / f"{op_func_name}.{op_overload_name}.json"
+    if not op_conf.exists():
+        return None
+
+    with open(op_conf) as f:
```

Comment:
perhaps we can leverage the `LocalCache` class in the codecache.py for this?

Reply:
It depends on how to design the `key`. Will the hash of the content be the key? Currently, the cache is loaded at the initialization stage for a given operation. I'm not sure its performance benefits.